common/lib/xmodule/xmodule/tests/__init__.py

```diff
@@ -59,6 +61,8 @@ def __init__(self, **kwargs):
         kwargs.setdefault('id_reader', id_manager)
         kwargs.setdefault('id_generator', id_manager)
         kwargs.setdefault('services', {}).setdefault('field-data', DictFieldData({}))
+        self.tracker = DjangoTracker()
+        tracker.register_tracker(self.tracker)
```

Comment:
mocking the `tracker` should be sufficient

You shouldn't need these lines at all:

```python
self.tracker = DjangoTracker()
tracker.register_tracker(self.tracker)
```